jscomp/main/jsoo_main.ml

```diff
@@ -54,52 +56,44 @@ let implementation ~use_super_errors impl str : Js.Unsafe.obj =
   (* Question ?? *)
   (* let finalenv = ref Env.empty in *)
   let types_signature = ref [] in
-  if use_super_errors then (
-    Misc.Color.setup (Some Always);
-    Lazy.force Super_main.setup);
-
   try
-    Js_config.jsx_version := 3;
     (* default *)
     let ast = impl (Lexing.from_string str) in
-    let ast = Ppx_entry.rewrite_implementation ast in
+    let ast = Melange_ppx_lib.Ppx_entry.rewrite_implementation ast in
```

Comment:
@anmonteiro I am using this line with the intention of being able to process `bs.raw` and such extension nodes. But it doesn't seem to work:

```
$ node
> require('./_build/default/jscomp/main/jsoo_main.bc.js');
{}
> require('./stdlib.js');
{}
> ocaml.compile("[%%bs.raw \"var a = 1\"]");
{
  js_error_msg: "Line 1, 3:\n  Error Uninterpreted extension 'bs.raw'.",
  row: 0,
  column: 3,
  endRow: 0,
  endColumn: 9,
  text: "Uninterpreted extension 'bs.raw'.",
  type: 'error'
}
```

Do you have any ideas what I might be missing?

Reply:
Oh, this is the "other" part of the ppx, that processes the attributes!

```
> ocaml.compile(`type person\nexternal john : person = "john" [@@bs.module "MySchool"]`);
{
  js_code: '// Generated by Melange\n' +
    "/* This output is empty. Its source's type definitions, externals and/or unused code got optimized away. */\n"
}
```

I need to somehow plug in the actual `melange.ppx` that is using the ppxlib driver.

Reply:
Fixed in https://github.com/melange-re/melange/pull/538/commits/08be8255e8fc4f8955d4c501f3d1f2b04f1aa2fb.